### ISSUE TITLE
Select component always display a ellipse though it's width is big enough.

### ISSUE THREAD

**Issue by valleygtc** [ant-design/ant-design#22196]
- [ ] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[![Edit on CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/antd-reproduction-template-vxb5z)

### Steps to reproduce
You can just use the first demo code in the [doc](https://ant.design/components/select-cn/), open it with firefox dev-edition:

就直接使用官网上第一段 demo 代码，[这个](https://ant.design/components/select-cn/)，用火狐开发者版本浏览器打开，就能看到

```javascript
import React from "react";
import ReactDOM from "react-dom";
import "antd/dist/antd.css";
import "./index.css";

import { Select } from "antd";

const { Option } = Select;

function handleChange(value) {
  console.log(`selected ${value}`);
}

ReactDOM.render(
  <div>
    <Select defaultValue="lucy" style={{ width: 120 }} onChange={handleChange}>
      <Option value="jack">Jack</Option>
      <Option value="lucy">Lucy</Option>
      <Option value="disabled" disabled>
        Disabled
      </Option>
      <Option value="Yiminghe">yiminghe</Option>
    </Select>
    <Select defaultValue="lucy" style={{ width: 120 }} disabled>
      <Option value="lucy">Lucy</Option>
    </Select>
    <Select defaultValue="lucy" style={{ width: 120 }} loading>
      <Option value="lucy">Lucy</Option>
    </Select>
    <Select defaultValue="lucy" style={{ width: 120 }} allowClear>
      <Option value="lucy">Lucy</Option>
    </Select>
  </div>,
  document.getElementById("root")
);
```

### What is expected?
If Select component if wide enough, it should display all text.

如果 Select 组件的宽度足够，那么应该显示出所有文字。

### What is actually happening?
though Select component is wide enough, it display a small range of text and ellipse.

即使 Select 组件的宽度足够，也只会显示一部分文字，其余的都省略了。

| Environment | Info |
|---|---|
| antd | 4.0.2 |
| React | 16.13.0 |
| System | Windows |
| Browser | firefox dev-edition 75.0b2 (64 位)  |

**Comment by valleygtc** [comment]
image:

![图片](https://user-images.githubusercontent.com/25948267/76608301-5c62f680-6550-11ea-898e-adf98660a6e3.png)

**Comment by afc163** [comment]
Fixed in #22101, will be released in next patch.

**Comment by valleygtc** [comment]
chrome is ok. only firefox dev editon.

chrome 浏览器没这个问题，就是火狐开发者版本
